dependencies.yml

```diff
@@ -114,6 +114,7 @@ com.linecorp.armeria:
     - https://www.javadoc.io/doc/com.linecorp.armeria/armeria-javadoc/1.14.1/
 
 com.puppycrawl.tools:
+  # Requires jdk 11
   checkstyle: { version: '9.2.1' }
```

Comment:
Should we disable `checkstyle` task if the build JDK is 8?
https://github.com/line/centraldogma/blob/2ee824a003d1185643a4eadd6dddecfc336fca97/.github/workflows/gradle.yml#L51